### PR TITLE
Rules for greenwave.

### LINKED ISSUE #242: greenwave rules are missing
I use FMN to get notified about anything regarding "taskotron", and exclude unwanted services (like koji, etc). Yesterday I started receiving tens of greenwave notifications:

```
<fedora-notif> greenwave.greenwave.decision.update --  4 hours ago     ( triggered by https://apps.fedoraproject.org/notifications/kparal.id.fedoraproject.org/irc/32393 )
<fedora-notif> greenwave.greenwave.decision.update --  4 hours ago     ( triggered by https://apps.fedoraproject.org/notifications/kparal.id.fedoraproject.org/irc/32393 )
<fedora-notif> greenwave.greenwave.decision.update --  4 hours ago     ( triggered by https://apps.fedoraproject.org/notifications/kparal.id.fedoraproject.org/irc/32393 )
<fedora-notif> greenwave.greenwave.decision.update --  4 hours ago     ( triggered by https://apps.fedoraproject.org/notifications/kparal.id.fedoraproject.org/irc/32393 )
<fedora-notif> greenwave.greenwave.decision.update --  4 hours ago     ( triggered by https://apps.fedoraproject.org/notifications/kparal.id.fedoraproject.org/irc/32393 )
<fedora-notif> greenwave.greenwave.decision.update --  3 hours ago     ( triggered by https://apps.fedoraproject.org/notifications/kparal.id.fedoraproject.org/irc/32393 )
<fedora-notif> greenwave.greenwave.decision.update --  3 hours ago     ( triggered by https://apps.fedoraproject.org/notifications/kparal.id.fedoraproject.org/irc/32393 )
<fedora-notif> greenwave.greenwave.decision.update --  3 hours ago     ( triggered by https://apps.fedoraproject.org/notifications/kparal.id.fedoraproject.org/irc/32393 )
<fedora-notif> greenwave.greenwave.decision.update --  3 hours ago     ( triggered by https://apps.fedoraproject.org/notifications/kparal.id.fedoraproject.org/irc/32393 )
...
```

But I can't blacklist it, because greenwave rules are not present in FMN. Please add them (soon), thanks a lot.

### PR DESCRIPTION
**Pull request by ralphbean**
Fixes #242.

Signed-off-by: Ralph Bean <rbean@redhat.com>

### CODE REVIEW

File: fmn/tests/rules/test_greenwave.py [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+    }
+}
+
+
+class TestGreenwaveDecisionUpdate(Base):
+    '''Test greenwave_decision_update()'''
+
+    def setUp(self):
+        super(TestGreenwaveDecisionUpdate, self).setUp()
+        self.msg = copy.deepcopy(GREENWAVE_MSG)
```
Comment (jeremycline) [inline fmn/tests/rules/test_greenwave.py]: I don't think any rule should ever mutate the message so I'd recommend dropping this. This way if a rule _does_ mutate the message tests will fail.

**Review by jeremycline — CHANGES_REQUESTED** [review]
If you could rebase (there's a fix for the failing CI in develop now) and drop that deep copy I think this will be good to go.

Thanks!

File: fmn/tests/rules/test_greenwave.py (hunk 1 above)
Comment (ralphbean) [reply fmn/tests/rules/test_greenwave.py]: > I don't think any rule should ever mutate the message so I'd recommend dropping this. This way if a rule does mutate the message tests will fail.

Ah, but some of the *tests* here do mutate the message.

**Comment by ralphbean** [comment]
@jeremycline, ready for review again.

**Review by jeremycline — APPROVED** [review]

**Pull request merged**